### DISCUSSION TITLE
Only old firmware for Sequre SQ-001

### DISCUSSION THREAD

**Discussion by v4ng3n** [Ralim/IronOS#1034] [Q&A]
Hi,
I've got a Sequre SQ-001, and the latest firmware version I can get to work is v2.12-beta.
Is that normal? I thought the electronics of the Sequre was exactly the same as the TS100.
Also, when using the iron, I often have to hold down the button closest to the tip for the iron to keep at a high temperature.

**Comment by Ralim** [comment]
So the Sequre is a _clone_ device. So cant really promise it will work (I don't have any way to test). But that said; it _should_ work without issues.
What happens when you try the newer firmware images?
Holding the button at the front is entering boost mode and setting your temp up to like 400C or so, what setpoint temperature are you using?

**Comment by v4ng3n** [comment]
Thanks for answering.

When I try to copy over the hex-file, the copy-window pops up for a second,
but then I just get an error message saying "An unexpected error is keeping
you from copying the file". Error code is 0x800701B1 if that helps.
I’ve literally just tried that 10 times with v2.15.
With v2.12-beta it happened once, but the second time the process completed.

Is it the boost temp you're thinking of? I've set that to 450.
_

Best regards:

Anders Vangsted

Tlf.: +47 941 72 387
Mail: ***@***.***


Den ons. 1. sep. 2021 kl. 10.57 skrev Ben V. Brown ***@***.***

**Reply by Ralim** [reply]
So if the issue is copying the file over, that is less likely to be hardware differences and more about your setup / luck.
(1) Make sure you are extracting the files from the zip before copying
(2) Make sure that you have no antivirus that does "on access" scanning
(3) Try different usb ports
(4) Try a different computer

Sadly, its not really possible to support the Miniware USB bootloader (which I believe Sequre just copied the binary for), as we dont have any source code to it, and its known to contain bugs.

**Comment by v4ng3n** [comment]
Yeah, I see. I just thought it was a bit strange since the v2.12-beta
flashed with ease.

(1) I am.
(2) I wouldn’t think I have. And can’t we conclude that that’s not the
problem since v2.12-beta works?
(3) I already tried all USB-ports on my PC.
(4) I will try my girlfriend’s computer when we get home from job.

I thought about installing the alternative bootloader, but since I’m
considering myself as "one of those that don’t know what they’re doing", I
didn’t do it. 😅
But do you think that will make any difference? As of now I'm on bootloader
version 3.45. Do you know if there's a newer version I can install?

 _

Best regards:

Anders Vangsted

Tlf.: +47 941 72 387
Mail: ***@***.***


Den tor. 2. sep. 2021 kl. 01.34 skrev Ben V. Brown ***@***.***

**Reply by Ralim** [reply]
(2) Generally not 😂 Since the bootloader only sees it as a hex file, so long as its a valid hex file there isnt really much we can "change" to to cause it to work or not. if the file is _invalid_ you will get the .ERR file. Where as your error indicates something going wrong in the USB/Filesystem layer which is well outside of the code we can control here.
(3) Have you tried different cables?
(4) Would be good if you can 😓 Realise its an ask

Honestly, probably best not to flash it if you have never used USB-DFU command line tools before, as it just opens a new can of worms :P 
But if you get desperate you can. 

There is no newer version, 3.43 is the only one I have seen on any of my TS100 units. The TS80(P) use newer ones but im not _certain_ if they are backwards flashable

**Comment by paulfertser** [comment]
On Thu, Sep 02, 2021 at 01:15:36AM -0700, Vangen92 wrote:
> (2) I wouldn’t think I have. And can’t we conclude that that’s not the
> problem since v2.12-beta works?

Flashed firmware doesn't anyhow affect the vendor bootloader. At
all. No matter what data is there in the application area the
bootloader is going to work the same. If you can't use it now it means
something else about your system changed.

**Comment by v4ng3n** [comment]
(3) Actually not. I just concluded that my method and hardware wasn't the
problem since I could flash v2.12-beta. But I will try another cable too.
(4) No ask at all. I'm just glad you want to help! 😊

I haven't, so probably I should just stay on the stock version. 😅 Unless
you think it might make a difference?
Honestly, I don't really care to be on the latest and greatest version, as
long the version I'm on works for me. And the biggest problem is that the
iron doesn't heat up, unless I keep it in boost mode by holding down the
button.
_

Best regards:

Anders Vangsted

Tlf.: +47 941 72 387
Mail: ***@***.***


Den tor. 2. sep. 2021 kl. 10.47 skrev Ben V. Brown ***@***.***

**Comment by v4ng3n** [comment]
Hi Ben and Poul,

Sorry for the late answer. I totally forgot about this until I had to use
the iron again.
As suggested, I tried my girlfriends PC, and it worked with v2.15 in first
try!
I don't know why it wouldn't work on mine, but at least it does now.

Thanks for the help!
_

Best regards:

Anders Vangsted

Tlf.: +47 941 72 387
Mail: ***@***.***


Den tor. 2. sep. 2021 kl. 11.04 skrev Anders Vangsted <
***@***.***>:

**Comment by v4ng3n** [comment]
Hi again,

So, as I wrote before I got the v2.15 flashed to my iron, and I thought
everything was good. But then when I tried to use it, it wouldn’t rise in
temperature. It just stood still at 29 degrees Celsius. I tried boost mode,
I tried changing some settings, I tried the reset function, and I tried
calibrating the temperature, but nothing helped. So, I thought “Screw
this”, and reinstalled v2.12-beta, but to my big surprise the same thing
happened…

After some irritation I tried to install the stock firmware, and when I did
the iron had some flickering in the temperature, and then read “SEN ERR”. I
made a quick search online, and found one thread saying, that they had to
get a new tip because of this sensor error. Therefore, I opened up the
iron, and found that the two brackets holding the tip in place was totally
loose. I tightened them down, and then there was no sensor error anymore,
and the iron worked just fine. Then I reinstalled v2.12-beta, and now that
also works just fine.

I think I might reinstall v2.15 when my girlfriend gets back home with her
computer, but I’m not quite sure as a fear of something else going wrong.

I don’t know if this will be helpful in any ways, but it might and now you
know.

_

Best regards:

Anders Vangsted

Tlf.: +47 941 72 387
Mail: ***@***.***


Den tir. 7. sep. 2021 kl. 15.35 skrev Anders Vangsted <
***@***.***>:
